src/csharp/README.md

```diff
@@ -4,10 +4,13 @@ gRPC C#
 
 A C# implementation of gRPC.
 
-Status
-------
+SUPPORTED PLATFORMS
+------------------
+
+- .NET Framework 4.5+ (Windows)
+- [.NET Core](https://dotnet.github.io/) on Linux, Windows and Mac OS X (starting from version 1.0.1)
+- Mono 4+ on Linux, Windows and Mac OS X
```

Comment:
nit: I know tests of the .NET core target are under actual .NET core platform, but should we refer to more general netstandard platform here, in  case one wants to consume it from something that's not .NET core but netstandard1.5-compatible?